source/adminguide/service_offerings.rst

```diff
@@ -721,3 +721,53 @@ default system offering used for System VMs.
 
 #. Destroy the existing CPVM or SSVM offerings and wait for them to be
    recreated. The new CPVM or SSVM are configured with the new offering.
+
+
+Changing the Default System Offering for Virtual Routers
+---------------------------------------------------
+
+As a CloudStack administrator, you can change the default system
+offering used for Virtual Routers.
+
+#. Create a new system offering
+
+   For more information, see `“Creating a New System Offering”
+   <service_offerings.html#creating-a-new-system-service-offering>`_.
+
+#. (Optional) Create a new network offering with SystemOffering
+
+   For more information, see `“Creating a New Network Offering”
+   <networking.html#creating-a-new-network-offering>`_.
+
+#. (Optional) Change account setting
+
+   You can change the default system offering for Virtual Routers of a particular
+   account by changing the account's setting "router.service.offering" to the uuid
+   of the system offering.
+
+   For more information, see `“Setting Local Configuration Parameters”
+   <../installguide/configuration.html#setting-local-configuration-parameters>`_.
+
+#. (Optional) Change global configuration
+
+   You can change the default system offering for Virtual Routers of all accounts
+   by changing the global configuration "router.service.offering" to the uuid of the system offering.
+
+   For more information, see `“Setting Global Configuration Parameters”
+   <../installguide/configuration.html#setting-global-configuration-parameters>`_.
+
+When you create a network, the virtual routers will use the system offering in their NetworkOffering.
```

Comment:
```suggestion
When you create a network, the virtual routers will use the system service offering in their Network Offering.
```

Reply:
I don't agree with this, either we should use camel case or we should use english concepts, not a mix of both.